andela/team-odd-bn-backend
# #169258635 Add profile page settings

#### What does this PR do?
Add user profile page settings
#### Description of Task to be completed?
- This PR adds a route for the user profile page
- This PR adds the capabilities for users to update their profile after signup is done
- This PR adds the capabilities for users to view their profile
#### How should this be manually tested?
- Open your terminal
- Run `git clone https://github.com/andela/team-odd-bn-backend.git`
- Run`cd team-odd-bn-backend`
- Run `git checkout -b ft-user-profile-settings-169258635`
- Run `npm install`
- update `.env` file with super admin credentials
- Run `npm run dev-start` check API in development 
- Run `npm test` testing API
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#169258635](https://www.pivotaltracker.com/story/show/169258635)
#### Screenshots (if appropriate)
<img width="751" alt="Screen Shot 2019-11-22 at 06 45 57" src="https://user-images.githubusercontent.com/35301129/69398656-d64a4800-0cf3-11ea-930a-a970ebb03ab0.png">
<img width="758" alt="Screen Shot 2019-11-22 at 06 48 17" src="https://user-images.githubusercontent.com/35301129/69398716-1f9a9780-0cf4-11ea-8261-872c10396861.png">
<img width="740" alt="Screen Shot 2019-11-24 at 12 07 58" src="https://user-images.githubusercontent.com/35301129/69493248-23c1e300-0eb5-11ea-83dd-e142b4796075.png">
<img width="1421" alt="Screen Shot 2019-11-24 at 12 33 34" src="https://user-images.githubusercontent.com/35301129/69493359-b57e2000-0eb6-11ea-9058-bb8963b21bce.png">
<img width="1421" alt="Screen Shot 2019-11-20 at 17 33 59" src="https://user-images.githubusercontent.com/35301129/69252626-0098ea00-0bbc-11ea-9c02-732385b2001d.png">
#### Questions:
N/A
